spec/support/shipping_options.rb

```diff
@@ -65,6 +65,12 @@ def invoice_form
       invoice_date: '20170816',
       reason_for_export: '',
       currency_code: 'USD',
+      discount: {
+        monetary_value: '5.00'
```

Comment:
For me, nested it here doesn't seem too practical. We're asserting the XML internals, i.e Schema name node into the properties were are expecting. I think just `discount: 1` would be enough IMO